include/oneapi/dpl/pstl/numeric_impl.h

```diff
@@ -194,7 +194,8 @@ __brick_transform_scan(_RandomAccessIterator __first, _RandomAccessIterator __la
 template <typename _Tp, typename _BinaryOperation>
 using is_arithmetic_udop =
     ::std::integral_constant<bool, ::std::is_arithmetic<_Tp>::value &&
-                                       !::std::is_same<_BinaryOperation, ::std::plus<_Tp>>::value>;
+                                       !::std::is_same<_BinaryOperation, ::std::plus<_Tp>>::value &&
```

Comment:
What's about `__dpl_sycl::__plus` ?

Reply:
This predicate implemented in our code as
```C++
#if _ONEDPL_SYCL2020_FUNCTIONAL_OBJECTS_PRESENT
template <typename _T = void>
using __plus = sycl::plus<_T>;
#else  // _ONEDPL_SYCL2020_FUNCTIONAL_OBJECTS_PRESENT
template <typename _T>
using __plus = sycl::ONEAPI::plus<_T>;
#endif // _ONEDPL_SYCL2020_FUNCTIONAL_OBJECTS_PRESENT
```

where ```sycl::plus<_T>``` implemented as
```C++
template <typename T = void> using plus = std::plus<T>;
```

Reply:
@MikeDvorskiy Fixed at https://github.com/oneapi-src/oneDPL/pull/1125/commits/3f3e0d908c7af03f870b5f7c5724558dc4b4fc19 : is it that you mean?